core/src/main/resources/jenkins/install/UpgradeWizard/footer_fr.properties

```diff
@@ -0,0 +1,3 @@
+msg.before=Bienvenue sur Jenkins 2!\u0020
```

Comment:
2 pais of eyes are better than one, good catch.
I correct this one by amending the commit.